clio-transfer-model/src/main/scala/org/broadinstitute/clio/util/json/JsonSchema.scala

```diff
@@ -111,6 +112,7 @@ object JsonSchema {
       case t if t =:= typeOf[Option[String]]         => (false, aString)
       case t if t <:< typeOf[Option[EnumEntry]]      => (false, aString)
       case t if t =:= typeOf[Option[OffsetDateTime]] => (false, aTime)
+      case t if t =:= typeOf[Option[UUID]]           => (false, aString)
```

Comment:
This is good enough for now, but if you want to geek out see ...
Wow, never mind. There is no JSON schema format for UUID.